docs/overview/contributing/the_basics.rst

```diff
@@ -479,15 +479,15 @@ With Docker
 
 #. Optional Flags: Various optional flags are available for running the tests such as :code:`device`, :code:`backend`, etc.
     1. :code:`device`: 
-        1. This flag enables setting of the device where the tests would be run.
+        1. This flag enables the setting of the device where the tests would be run.
         2. Possible values being :code:`cpu` and :code:`gpu`.
         3. Default value is :code:`cpu`
     2. :code:`backend`:
         1. This flag enables running the tests for particular backends.
-        2. The values of this flag could be any possible combination of JAX, numpy, tensorflow and torch.
+        2. The values of this flag could be any possible combination of JAX, numpy, tensorflow, and torch.
         3. Default value is :code:`jax,numpy,tensorflow,torch`.
     3. :code:`num-examples`:
-        1. Set the maximum number for examples to be generated by Hypothesis.
+        1. Set the maximum number of examples to be generated by the Hypothesis.
```

Comment:
Hypothesis is a library here, not a noun.
```suggestion
        1. Set the maximum number of examples to be generated by Hypothesis.
```

Reply:
Ok,👍